py/ig_compress_equal_clusters.py

```diff
@@ -81,18 +81,19 @@ def check_fa_rcm_consistency(fa_filename, rcm_filename):
 
     if (not args.tmp_fa_file):
         args.tmp_fa_file = tempfile.mkstemp(suffix=".fa", prefix="igrec_")[1]
-    if (not args.tmp_map_file):
-        args.tmp_map_file = tempfile.mkstemp(suffix=".map", prefix="igrec_")[1]
+    if (not args.tmp_rcm_file):
+        args.tmp_rcm_file = tempfile.mkstemp(suffix=".rcm", prefix="igrec_")[1]
 
     print "Command line: %s" % " ".join(sys.argv)
 
     home_directory = os.path.abspath(os.path.dirname(os.path.realpath(__file__))) + '/../'
     run_trie_compressor = os.path.join(home_directory, 'build/release/bin/./ig_trie_compressor')
-    cmd_line = "%s -i %s -o %s -m %s" % (run_trie_compressor, args.input, args.tmp_fa_file, args.tmp_map_file)
+    cmd_line = "%s -i %s -o %s -m %s" % (run_trie_compressor, args.input, args.tmp_fa_file, args.tmp_rcm_file)
     os.system(cmd_line)
 
-    with open(args.tmp_map_file) as fin:
-        input_read_num2compressed_cluster = [int(line.strip()) for line in fin]
+    with open(args.tmp_rcm_file) as fin:
+        # obtain simple map from rcm file
+        input_read_num2compressed_cluster = [int(line.strip().split("\t")[1]) for line in fin]
```

Comment:
Lines in RCM may have an arbitrary order! We should parse and use IDs here instead of relying on ordering